src/fq_default_mat.h

```diff
@@ -1134,28 +1134,29 @@ FQ_DEFAULT_MAT_INLINE int fq_default_mat_inv(fq_default_mat_t B,
 
 /* Solving *******************************************************************/
 
-FQ_DEFAULT_MAT_INLINE slong fq_default_mat_rref(fq_default_mat_t A,
+FQ_DEFAULT_MAT_INLINE slong fq_default_mat_rref(fq_default_mat_t B, const fq_default_mat_t A,
```

Comment:
This needs to be changed in the documentation as well.

Reply:
Good catch!